docs/proposals/20220330-topology-mutation-hook.md

```diff
@@ -138,7 +141,7 @@ Once the extension is registered the discovery hook is called and the Extension
 
 ### ClusterClass author guide
 
-A ClusterClass author can use an External Patch Extension by referencing it in a ClusterClass and adding the corresponding variable definitions.
+A ClusterClass author can use an External Patch Extension by referencing it in a ClusterClass and adding the corresponding variable definitions, or relying on variable definitions supplied by a Variable Discovery Hook.
```

Comment:
I think this won't work anymore: "adding the corresponding variable definitions"